decidim-core/app/controllers/concerns/decidim/amendments_controller.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module Decidim
+  class AmendmentsController < Decidim::ApplicationController
+    include FormFactory
+    before_action :authenticate_user!
+    helper_method :amendable
+
+    def new
+      @form = form(Decidim::CreateAmendForm).from_params(params)
```

Comment:
Maybe use `from_model` with `amendable`?

Reply:
:+1:  Thaks fro the tip